packages/editor/src/store/effects.js

```diff
@@ -104,7 +104,7 @@ export function selectPreviousBlock( action, store ) {
 
 	const firstRemovedBlockClientId = action.clientIds[ 0 ];
 	const state = store.getState();
-	const currentSelectedBlock = getSelectedBlock( state );
+	const currentSelectedBlock = getSelectedBlockClientId( state );
```

Comment:
I feel this is a bug fix somehow because we're comparing the previous result with a clientId and not a block object.

Reply:
> I feel this is a bug fix somehow because we're comparing the previous result with a clientId and not a block object.

I agree. It's probably not made any difference only because `selectBlock` won't do anything if the ID hasn't changed. Maybe it's useless code? Can be good to avoid dispatching if we don't need it.

Perhaps pull out to a separate pull request?